### ISSUE TITLE
Do a version bump.

### ISSUE THREAD

**Issue by emilio** [servo/string-cache#184]
How should a version bump be done for this crate after #183?

My idea is:

 * Breaking version bump of `string-cache-codegen`, given the new `hashes` array is written from there and it wouldn't be compatible with older versions of `string-cache`.
 * Making `string-cache` depend on the newer `string-cache-codegen` version, given the new field is just public so it can be accessed from macros, not part of the public API.
 * Minor bump on that.

But I guess @nox may have different opinions? Should we do a breaking bump everywhere?

**Comment by SimonSapin** [comment]
I think the problem is that for users who depend on current versions of `string-cache` and `string-cache-codegen`, when they run `cargo update` or build without a `Cargo.lock` file they would get the new version on `string-cache` (if we make it a non-breaking number) but not of `string-cache-codegen`. That the former depends on a different version of the latter doesn’t matter, Cargo would happily have two versions in the build. Then things would break because the old codegen crate doesn’t generate the new `hashes` field.

**Comment by nox** [comment]
I'm leaning towards making a breaking bump on both, because bumping only the codegen crate would break a fresh build of current `html5ever-atoms`.

If we do a breaking bump on both, let's seize this opportunity to remove the dependency on `rustc-serialize`.

**Comment by emilio** [comment]
Done that way in #185
